pxr/imaging/hdSt/material.cpp

```diff
@@ -167,6 +176,72 @@ HdStMaterial::_ProcessTextureDescriptors(
                                        doublesSupported);
 }
 
+#ifdef PXR_MATERIALX_SUPPORT_ENABLED
+void
+HdStMaterial::StartEarlyInit(
+    HdSceneDelegate *sceneDelegate,
+    HdSt_MaterialXSyncDispatcher *syncDispatcher)
```

Comment:
The `syncDispatcher` is the global cache of parallel tasks, deduplicating them by hash and implementing waiting for them.